app/models/manageiq/providers/embedded_terraform/provider/default_terraform_objects.rb

```diff
@@ -0,0 +1,27 @@
+module ManageIQ::Providers::EmbeddedTerraform::Provider::DefaultTerraformObjects
```

Comment:
What is the purpose of these default_terraform_objects with respect to syncing terraform templates?  I don't see any callers in this PR.

Reply:
I think this is copy&paste from embedded ansible, it isn't necessary here and should be deleted https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/embedded_ansible/provider/default_ansible_objects.rb

Reply:
Needs review.